backends/platform/ios7/ios7_options.cpp

```diff
@@ -91,17 +64,33 @@ void IOS7OptionsWidget::handleCommand(GUI::CommandSender *sender, uint32 cmd, ui
 	switch (cmd) {
 	case kHelpCmd: {
 		GUI::MessageDialog help(_(
-"Gestures and controls:
-
-One finger tap: Left mouse click
-Two finger tap: Right mouse click
-Two finger double tap: ESC
-Two finger swipe (bottom to top): Toggles Click and drag mode
-Two finger swipe (left to right): Toggles between touch direct mode and touchpad mode
-Two finger swipe (top to bottom): Global Main Menu
-Three finger swipe: Arrow keys
-Pinch gesture: Enables/disables keyboard
-Keyboard spacebar: Pause"));
+#if TARGET_OS_IOS
+			"Gestures and controls:\n"
```

Comment:
You must wrap both strings into _(), so they are picked up by gettext

and add this file to POTFILES